x/accounts/genesis.go

```diff
@@ -80,7 +80,14 @@ func (k Keeper) ImportState(ctx context.Context, genState *v1.GenesisState) erro
 }
 
 func (k Keeper) importAccount(ctx context.Context, acc *v1.GenesisAccount) error {
-	// TODO: maybe check if impl exists?
+	isRegistered, err := k.isAccountTypeRegistered(ctx, acc.AccountType)
+	if err != nil {
+		return err
+	}
```

Comment:
I would prefer to
``` 
         if isRegistered, err := k.isAccountTypeRegistered(ctx, acc.AccountType); err != nil {
                  log.Fatalf("isAccountTypeRegistered Error: %v", err)
		return err
	}
```

Reply:
Thanks for the review. 
I aim to maintain a similar format to other source codes, as it enhances readability of the code.